interfaces/system_key_test.go

```diff
@@ -374,6 +486,8 @@ func (s *systemKeySuite) TestSystemKeysUnmarshalSame(c *C) {
 		],
 		"apparmor-parser-features": [],
 		"apparmor-parser-mtime": 1589907589,
+		"apparmor-prompting-supported": %s,
+		"apparmor-prompting-supported-and-enabled": %s,
```

Comment:
You can use the `%t` format specifiers here
```suggestion
		"apparmor-prompting-supported": %t,
		"apparmor-prompting-supported-and-enabled": %t,
```